src/main/java/frc/team2412/robot/Controls.java

```diff
@@ -41,4 +59,9 @@ private void bindDrivebaseControls() {
 		driveController.start().onTrue(new InstantCommand(s.drivebaseSubsystem::resetGyro));
 		driveController.rightStick().onTrue(new InstantCommand(s.drivebaseSubsystem::toggleXWheels));
 	}
+
+	private void bindLauncherControls() {
+		launcherPodiumPresetButton.onTrue(new SetAngleLaunchCommand(s.launcherSubsystem, 3392, 75));
```

Comment:
> `3392, 75`

Would prefer to have constants for these / reuse the existing ones in `LauncherSubsystem`.